config-template.yaml

```diff
@@ -28,10 +28,14 @@ singleuser:
   startTimeout: 180
 
 auth:
+  type: custom
```

Comment:
Since I noticed that the `FirstUseAuthenticator` wasn't properly storing/checking the passwords and just letting anyone log into an account no matter what the password, I quickly had to debug this. So I _both_ installed the `jupyterhub-firstuseauthenticator` package and added this line here. I'm assuming this line fixed it though, because I don't see how my local environment could affect anything. 😅